cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TransactionsCommandTest.scala

```diff
@@ -411,5 +415,107 @@ All users may terminate their own currently executing transactions.
         }
       }
     }
+    section("Combining transaction commands", id="query-combine-tx-commands") {
+      p(
+        """
+In difference to other show commands, the `SHOW` and `TERMINATE TRANSACTIONS` commands can be combined in the same query.
+
+[NOTE]
+====
+When combining multiple commands the `YIELD` and `RETURN` clauses are mandatory and must not be omitted.
+In addition, the `YIELD` clause needs to explicitly list the yielded columns. `YIELD *` is not permitted.
```

Comment:
```suggestion
In addition, the `YIELD` clause needs to explicitly list the yielded columns. 
`YIELD *` is not permitted.
```

Reply:
```suggestion
In addition, `YIELD *` is not permitted instead the `YIELD` clause needs to explicitly list the yielded columns.
```

Reply:
I think your suggestion makes the sentence too long. The original one was already good, I just put the second sentence in another line as it's supposed to be when you finish a sentence.

Reply:
Which made me notice I wasn't really happy about it being 2 sentences... It felt like it was cut up between reason and consequence when they belong together

Reply:
I see. But now the word "instead" looks a bit lost in the middle of the sentence. I would suggest any of these possibilities then, in case you want to keep it as a whole sentence or divide it two:

```suggestion
`YIELD *` is not permitted. 
Instead, the `YIELD` clause needs to explicitly list the yielded columns.
```
or
```suggestion
Since `YIELD *` is not permitted here, the `YIELD` clause needs to explicitly list the yielded columns.
```

Reply:
in general I like the second one better, but I feel like `since` is a bit odd as this is the first mention of it being disallowed. Maybe having two sentences but flipping the order works (and making the `YIELD *` part into a bit more sentencey sentence :P)?

```suggestion
In addition, `YIELD *` is not permitted.
Instead, the `YIELD` clause needs to explicitly list the yielded columns.
```

Reply:
Yep, we can go with that too! :)